orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/config/KubernetesPreconfiguredJobProperties.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.config;
+
+import io.kubernetes.client.models.V1Job;
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Data
+public class KubernetesPreconfiguredJobProperties extends PreconfiguredJobStageProperties {
+
+  public String account;
+  public String application;
+  public V1Job manifest;
+
+  public List<String> getAdditionalFields() {
+    return new ArrayList<>(Arrays.asList("account", "manifest", "application"));
```

Comment:
This wrapping of the `asList` result is unnecessary. If you're concerned about a consumer modifying it, then use `ImmutableList.of()`.